build-info/github.com/java-json-tools/json-patch/build.yaml

```diff
@@ -5,3 +5,7 @@ repositories:
 additionalArgs:
   - "-x"
   - "checkSigningRequirements"
+preBuildScript: |
+  sed -i /options.addStringOption/d build.gradle
+  sed -i /.*propdeps.*/d build.gradle
+  sed -i -e 's/provided/compileOnly/' project.gradle
```

Comment:
The prop-deps plugin isn't available anymore in any spring repo (apart from some snapshot versions but even they don't resolve correctly). By removing it and replacing its definition of `provided` -> `compileOnly` (which is Gradle's closest equivalent) the build can proceed.
(cc @dwalluck re prop-deps)